pyproject.toml

```diff
@@ -30,7 +30,7 @@ dependencies = [
   # changing the version
   # (plugins.mkdocstrings.handlers.python.import)
   "frequenz-client-microgrid >= 0.4.0, < 0.5.0",
-  "frequenz-channels >= 1.0.0-rc1, < 2.0.0",
+  "frequenz-channels >= 1.1.0, < 2.0.0",
```

Comment:
You must be happy about finally getting rid of the `-rc1` :laughing:

Just as a heads up, once we reach *the real 1.0.0* :tm: for the SDK, we can't bump minimum dependencies anymore.

Also, while you are upgrading, wouldn't it make sense to also replace the `LatestValueCache` with the one in channels? But OK to do in a separate PR too.

Reply:
> You must be happy about finally getting rid of the -rc1 😆

properly happy. :-)

> Also, while you are upgrading, wouldn't it make sense to also replace the LatestValueCache with the one in channels? But OK to do in a separate PR too.

will do it separately.